exercises/en/test_02_18.py

```diff
@@ -0,0 +1,13 @@
+def test():
+    # Here we can either check objects created in the solution code, or the
+    # string value of the solution, available as __solution__. A helper for
+    # printing formatted messages is available as __msg__. See the testTemplate
+    # in the meta.json for details.
+
+    # If an assertion fails, the message will be displayed
+    assert "data/pokemon.csv" in __solution__ , "Are you specifying the correct data path?"
+    assert "index_col=0" in __solution__ , "Are you making sure to add index_col=0 ?"
+    assert  min(list(mighty_pokemon.defense)) == 101, "Are you selecting pokemons with defense > 100?"
```

Comment:
I'm wondering if there are any more tests you can do for this? Maybe sum the column of defense and  and another test for attack?